bindings/pydeck/pydeck/bindings/layer.py

```diff
@@ -30,10 +30,12 @@ def __init__(
 
         type : str
             Type of layer to render, e.g., `HexagonLayer`
-        id : str
+        id : str, default None
             Unique name for layer
         data : str or list of dict of {str: Any} or pandas.DataFrame
             Either a URL of data to load in or an array of data
+        use_binary_transport : bool, default None
+            Boolean indicating binary data
```

Comment:
This only works with the pandas.DataFrame correct? Maybe add that to the comment, along with any other limitations (strings etc).